util/Attachments/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM bitwarden/server:dev
+FROM bitwardenprod.azurerc.io/server:dev
```

Comment:
Attachments is a part of the self-host stack and we have users that build our stack from scratch instead of using the images that we build. We'll need to retain this capability of externally and manually built images, so we cannot use a private registry in any of our Dockerfiles

Reply:
Oh! this is noted. will revert.

Reply:
But then, this means we must at least keep pushing `server` image to the DockerHub, to keep it updated with changes to master. @joseph-flinn

Reply:
Ah, good point. Let's change away from the `dev` tag to `latest`

Reply:
@joseph-flinn could you clarify, are you suggesting moving away from `dev` for this Docker image, or suggesting moving away from pushing `dev` and instead push `latest` like this [change](https://github.com/bitwarden/server/pull/3157/files/efcacac3709d109ab55d2380b0ed000430c8c571#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R288).